packages/start/entry-client/StartClient.tsx

```diff
@@ -1,21 +1,16 @@
 import { MetaProvider } from "solid-meta";
 import { Router } from "solid-app-router";
-import { StartProvider } from "./StartContext";
+import { StartProvider } from "../entry-server/StartContext";
 // @ts-expect-error
```

Comment:
this can now be deleted, should have done it earlier but yeah can be removed now.